Documentation/gettingstarted/hubble.rst

```diff
@@ -96,18 +96,18 @@ Relay and Hubble's graphical UI.
 
                helm install cilium |CHART_RELEASE| \\
                   --namespace kube-system \\
-                  --set global.nodeinit.enabled=true \\
-                  --set global.kubeProxyReplacement=partial \\
-                  --set global.hostServices.enabled=false \\
-                  --set global.externalIPs.enabled=true \\
-                  --set global.nodePort.enabled=true \\
-                  --set global.hostPort.enabled=true \\
-                  --set global.pullPolicy=IfNotPresent \\
-                  --set config.ipam=kubernetes \\
-                  --set global.hubble.enabled=true \\
-                  --set global.hubble.listenAddress=":4244" \\
-                  --set global.hubble.relay.enabled=true \\
-                  --set global.hubble.ui.enabled=true
+                  --set nodeinit.enabled=true \\
+                  --set kubeProxyReplacement=partial \\
+                  --set hostServices.enabled=false \\
+                  --set externalIPs.enabled=true \\
+                  --set nodePort.enabled=true \\
+                  --set hostPort.enabled=true \\
+                  --set image.pullPolicy=IfNotPresent \\
+                  --set ipam.mode=kubernetes \\
+                  --set hubble.enabled=true \\
+                  --set hubble.listenAddress=":4244" \\
+                  --set hubble-relay.enabled=true \\
```

Comment:
Not fixed, this should be `hubble.relay`

Reply:
I have similar comment on differences between `hubble.reply.enabled` vs `hubble-replay.enabled` (similar for hubble-ui as well).

Reply:
there won't be any more hubble-relay subchart, which is the only possible way to have a top level key with a hyphen, as values do not support hyphenation in helm

all hubble values are now set under the top level `hubble:` map, with relay being set at `hubble.relay` and UI being set at `hubble.ui`